packages/cxl-ui/src/components/cxl-app-layout.js

```diff
@@ -26,7 +26,7 @@ export class CXLInstituteLayout extends LitElement {
   }
 
   set asideOpened(value) {
-    localStorage.setItem(ASIDE_LOCAL_STORAGE_KEY, JSON.stringify(value));
+    localStorage.setItem(this._asideLocalStorageKey, JSON.stringify(value));
```

Comment:
https://github.com/conversionxl/aybolit/pull/62#issuecomment-656087586 this accidentally remained?

Reply:
yes, my mistake